packages/linked-data-proof/src/__tests__/Ed25519Signature2018.digitalbazaar/__fixtures__/dereferencingDocumentLoader.ts

```diff
@@ -0,0 +1,77 @@
+import { contexts } from "./contexts";
+import controller from "./controller.json";
+import jsonld from "jsonld";
+
+const contextResolver = async (iri: string) => {
+  if (contexts[iri]) {
+    return { document: contexts[iri] };
+  }
+  return undefined;
+};
+
+const documentResolver = async (iri: string) => {
+  if (iri.startsWith(controller.id)) {
+    return {
+      documentUrl: controller.id,
+      document: controller
+    };
+  }
+  return undefined;
+};
+
+// Note: you implement this without using JSON-LD frame,
```

Comment:
lets address this comment, and not use frame.

Reply:
This has been removed.